packages/libthread-db/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://android.googlesource.com/platform/ndk/
+TERMUX_PKG_DESCRIPTION="Thread debugging library"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=22
+#TERMUX_PKG_SRCURL=https://android.googlesource.com/platform/ndk/+archive/refs/tags/ndk-r${TERMUX_PKG_VERSION}/sources/android/libthread_db.tar.gz
+TERMUX_PKG_SKIP_SRC_EXTRACT=true
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_configure() {
+	:
+}
+
+termux_step_make() {
+	$CC $CPPFLAGS -I. $CFLAGS libthread_db.c -shared -o libthread_db.so $LDFLAGS
+}
+
+termux_step_make_install() {
+	install -Dm600 -t $TERMUX_PREFIX/lib "$TERMUX_PKG_BUILDDIR/libthread_db.so"
+	ln -sf $TERMUX_PREFIX/lib/libthread_db.so $TERMUX_PREFIX/lib/libthread_db.so.1
```

Comment:
It would be great if you also install the header file for the package

Reply:
Fixed.